src/Microsoft.Graph/Microsoft.Graph.Beta.csproj

```diff
@@ -26,7 +26,9 @@
 - https://github.com/microsoftgraph/msgraph-beta-sdk-dotnet/blob/master/CHANGELOG.md
     </PackageReleaseNotes>
     <!-- edit this value to change the current major.minor.patch version -->
-    <VersionPrefix>5.78.0</VersionPrefix>
+     <!-- x-release-please-start-version -->
```

Comment:
We most likely need to remove the version suffix attribute as well

Reply:
Done